book/webspaces.rst

```diff
@@ -270,7 +270,7 @@ localization. The following fragment shows again how to this:
 
 .. code-block:: xml
 
-    <url language="de" country="at">www.example.org</url>
+    <url language="de" country="at">www.website.org</url>
```

Comment:
would still use `example.org` in the official examples as it is official domain special for this kind of things :)